text_report_generator.cpp

```diff
@@ -524,9 +524,9 @@ void TextReportGenerator::output(ostream& f, const json& report, bool show_regio
 
     bool show_unit_attitudes = false;
     if (report.contains("administrative")) {
-        if (!report["administrative"].contains("times_sent"))
+        if (!report["administrative"].contains("times_sent") || !report["administrative"]["times_sent"])
```

Comment:
I'd initially made the assumption here that the field wouldn't be output at all if it was false, but the actual generation
code always created the field and set it's value to either true or false.   For safety it will now check existence and/or the value being truthful.

In the case of times, if the value isn't set OR it's set to false then the times isn't being set.  For the password, if it's set AND it's set to true then we output the warning.

Found this while testing the new website with an downloaded report for a new faction where it had a password set..
This wasn't caught in unit tests or snapshot tests as they don't set faction passwords so they (as expected) had this warning.